src/ast/stmt/IfStmt.php

```diff
@@ -58,12 +58,8 @@ public function format(Parser $parser)
             $source .= $parser->indent();
             $source .= 'else';
             $source .= PHP_EOL;
-
             $parser->openScope();
-
-            $source .= $parser->indent();
```

Comment:
It was a bug. The next line was after responsible by taking care of indentation. With this line, the code formatting was:

```
if true
  do 1
else
    do 2
end
```
An extra indentation was being applied incorrectly.